### PR TITLE
20 icons

### PR DESCRIPTION
**Pull request by x9136**
## Type of change
:x: Bug fix (non-breaking change which fixes an issue)
:white_check_mark: Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:x: General change (non-breaking change that doesn't fit the above categories like copyediting)

## Icons added from icon request form

* DroidCam (`com.dev47apps.droidcam/com.dev47apps.droidcam.DroidCam`)
* DroidCamX (`com.dev47apps.droidcamx/com.dev47apps.droidcamx.DroidCamX`)
* eSewa (`com.f1soft.esewa/com.f1soft.esewa.activity.SplashScreenActivity`)
* Grammarly (`com.grammarly.android.keyboard/com.grammarly.host.SplashActivity`)
* iTAX (`com.ld.itax_pro_android/com.ld.itax_pro_android.MainActivity`)
* KBC Mobile (`com.kbc.mobile.android.phone.kbc/com.kbc.mobile.android.mbk.activities.SplashScreenActivity`)
* MIFARE Classic Tool (`de.syss.MifareClassicTool/de.syss.MifareClassicTool.Activities.MainMenu`)
* Polycam (`ai.polycam/ai.polycam.MainActivity`)
* SCB EASY (`com.scb.phone/com.scb.phone.view.activity.splash.SplashActivity`)
* Siddhartha BankSmart XP (`com.f1soft.banksmart.siddhartha/com.f1soft.bankxp.android.login.splash.SplashActivity`)
* Simply Wall St (`com.simplywallst.app/com.simplywallst.app.MainActivity`)
* Sky Map (`com.google.android.stardroid/com.google.android.stardroid.activities.SplashScreenActivity`)
* Smart Switch (`com.sec.android.easyMover/com.sec.android.easyMover.DistributionActivity`)
* Smart Switch (`com.sec.android.easyMover/com.sec.android.easyMover.DistributionNoIconActivity`)
* SmartThings (`com.samsung.android.oneconnect/com.samsung.android.oneconnect.ui.SCMainActivity`)
* Stay Awake (`org.hatapps.stayawake/com.example.myapplication2.MainActivity`)
* Transit (`com.thetransitapp.droid/com.thetransitapp.droid.shared.activity.SplashActivitytorontosubwayyellow`)
* Transit (`com.thetransitapp.droid/com.thetransitapp.droid.shared.activity.SplashActivity`)
* TrueMoney (`th.co.truemoney.wallet/th.co.truemoney.wallet.activity.LoadingScreenActivity`)
* Wolt (`com.wolt.android/com.wolt.android.activities.LauncherActivity`)
* XMind (`net.xmind.doughnut/net.xmind.doughnut.MainActivity`)
* Zipmex (`com.zipmex/com.zipmex.features.splash.LauncherEntryActivity`)

### CODE REVIEW

File: svgs/simply_wall_st.svg
Comment (Chefski) [inline svgs/simply_wall_st.svg:1]: 2 paths have a stroke of 11. Could you make it 10 or 12?

File: svgs/sky_map.svg
Comment (Chefski) [inline svgs/sky_map.svg:1]: Could you make this icon have a content area of 160x160.

File: svgs/transit.svg
Comment (Chefski) [inline svgs/transit.svg:1]: Could you make the content area 160x160px

**Review by Chefski — CHANGES_REQUESTED** [review]

File: svgs/transit.svg
Comment (x9136) [reply svgs/transit.svg:1]: The content area is 160x160px. Can you show me more specifically what's wrong?

Comment (Chefski) [reply svgs/transit.svg:1]: It's not quite at the border of 160x160
image

Comment (Chefski) [reply svgs/transit.svg:1]: I'm so stupid. I completely went autopilot here. Yeah its fine. I've gotten used to all icons being 160 on the width i forgot about the height

**Pull request merged**